File: PSG Tech/PSGLinkedList/src/taskid.java
import java.util.Scanner;

class Node2 {
    int data;
    Node next;

    Node2(int data) {
        this.data = data;
        this.next = null;
    }
}

class LinkedList2 {
    Node head;

    public void insertFront(int data) {
        Node newNode = new Node(data);

        newNode.next = head;
        head = newNode;
    }

    void display() {
        Node temp = head;

        while (temp != null) {
            System.out.print(temp.data);
            if (temp.next != null) {
                System.out.print(" ");
            }
            temp = temp.next;
        }
    }
}

public class taskid {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        LinkedList2 list = new LinkedList2();

        for (int i = 0; i < n; i++) {
            int data = sc.nextInt();
            list.insertFront(data);
        }

        list.display();
    }
}

File: Psgcas/taskid.py
class Node:
    def __init__(self,data):
        self.data = data
        self.next = None

class LinkedListProduct:
    def __init__(self):
        self.head = None

    def insert(self,data):
        newnode = Node(data)
        if self.head is None:
            self.head = newnode
            return

        newnode.next = self.head
        self.head = newnode

    def display(self):
        temp = self.head
        while temp:
            print(temp.data,end=" ")
            temp = temp.next

ll = LinkedListProduct()
n = int(input())
nums = list(map(int,input().split()))
for i in nums:
    ll.insert(i)

ll.display()